app/src/ui/preferences/appearance.tsx

```diff
@@ -30,20 +35,65 @@ export class Appearance extends React.Component<IAppearanceProps, {}> {
     } else {
       fatalError(`Unknown theme index ${index}`)
     }
+    this.props.onAutomaticallySwitchThemeChanged(false)
+  }
+
+  private onAutomaticallySwitchThemeChanged = (
+    event: React.FormEvent<HTMLInputElement>
+  ) => {
+    const value = event.currentTarget.checked
+
+    if (value) {
+      const usingDarkMode = isDarkModeEnabled()
```

Comment:
Since we're only using this variable in one place, we can just call `isDarkModeEnabled()` inline below.